docs/source/programming_resources/tutorial_specific/blocks/using_sensors/Using-Sensors-(Blocks).rst

```diff
@@ -1,5 +1,5 @@
-Using Sensors (Blocks)
-======================
+Using Sensors :bdg-warning:`Blocks`
+====================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".